src/motor_node.cc

```diff
@@ -486,12 +516,17 @@ int main(int argc, char* argv[]) {
             }
 
             // a periodic refresh of wheel type which is a safety net due to it's importance.
-            // This can be removed when a solid message protocol is developed
             if (robot->firmware_version >= MIN_FW_WHEEL_TYPE_THIN) {
                 // Refresh the wheel type setting
                 robot->setWheelType(robot->wheel_type);
                 mcbStatusPeriodSec.sleep();
             }
+            // a periodic refresh of drive type which is a safety net due to it's importance.
+            if (robot->firmware_version >= MIN_FW_DRIVE_TYPE) {
```

Comment:
I take it the firmware part of this can switch at any time without major effort and sending this at a high frequency won't cause any issues?

Reply:
This code along with other refresh next to it is gated to a lower rate.  See this block as it is using sysMaintPeriod.
This is non-ideal but I do this because I had seen it and could not figure it out so this is a 'just in case' workaround.
Non-ideal but too much to try to fix in this PR.